### ISSUE TITLE
[Bug]: Bismuth incorrectly tiles systemsettings and other applications even after setting  Minimum Geometry Size and is only fixed by manually reapplying tiling settings while the application is open.

### ISSUE THREAD

**Issue by respectablenoob-source** [Bismuth-Forge/bismuth#443]
### Summary

systemsettings dont tile correctly even after Setting Minimum Geometry Size unless tiling settings are replied while application is open.

### Steps to Reproduce

1.Enable tiling within bismuth
2.Set Minimum Geometry Size to 0(Tweaks section)
3.Launch systemsettings
4.systemsettings protudes out of screen/does not tile properly.

### Expected behavior

systemsettings would tile properly like all other applications(does not protude out of screen and tiles correctly without overlap)

### Bismuth version

3.1.4

### KDE Plasma version

5.25.5

### The platform KWin is running on

X11

**Comment by gikari** [comment]
Does applying [this](https://github.com/Bismuth-Forge/bismuth/issues/421#issuecomment-1284567506) window rule fix the issue for you?

**Comment by phrxmd** [comment]
I can reproduce this on Wayland and the window rule fix does not work for me. System settings will not allow itself to be smaller than 648x727 pixels, with Window Rules set up like this:

![Screenshot_20221020_123241-windowrules](https://user-images.githubusercontent.com/35058059/196925898-544ebe21-c263-4f3a-82a8-8f5bf470af49.png)

I wonder if the issue is unrelated to Bismuth though, because it happens also when System settings is floated.

**Comment by Aleksanaa** [comment]
> I can reproduce this on Wayland and the window rule fix does not work for me. System settings will not allow itself to be smaller than 648x727 pixels, with Window Rules set up like this: ![Screenshot_20221020_123241-windowrules](https://user-images.githubusercontent.com/35058059/196925898-544ebe21-c263-4f3a-82a8-8f5bf470af49.png) ("Erzwingen" means "force", "Anfangsplatzierung" means "initial placement", "Angeforderte Geometrie ignorieren" means "ignore requested geometry", "Minimale Größe" means "minimal size")
> 
> I wonder if the issue is unrelated to Bismuth though, because it happens also when System settings is floated.

Yes it's unrelated, but we may eventually find a solution so keeping this issue open is a good idea.
